src/models/sunburstChart.js

```diff
@@ -90,9 +76,9 @@ nv.models.sunburstChart = function() {
     //------------------------------------------------------------
 
     sunburst.dispatch.on('elementMouseover.tooltip', function(evt) {
-        evt['series'] = {
+        evt.series = {
             key: evt.data.name,
-            value: evt.data.size,
+            value: evt.data.value,
```

Comment:
You changed this, but this will break charts for people already using the chart and expecting size to work.   please do this for backward compatiblity:

    value:  evt.data.value || evt.data.size,